test/server_rendered_html_test.rb

```diff
@@ -61,15 +61,15 @@ def wait_to_ensure_asset_pipeline_detects_changes
 
         if WebpackerHelpers.available?
           new_file_path = '../dummy/app/javascript/components/NewList.js'
-          new_file_contents = <<~HEREDOC
-          var React = require("react")
-          module.exports = function() { return <span>"New List"</span> }
-          HEREDOC
+          new_file_contents = <<-HEREDOC
+var React = require("react")
+module.exports = function() { return <span>"New List"</span> }
+HEREDOC
         else
           new_file_path = '../dummy/app/assets/javascripts/components/ZZ_NewComponent.js.jsx'
-          new_file_contents = <<~HEREDOC
-          var NewList = function() { return <span>"New List"</span> }
-          HEREDOC
+          new_file_contents = <<-HEREDOC
+var NewList = function() { return <span>"New List"</span> }
```

Comment:
Seems like you might've messed up the indentation here

Reply:
The `<<-` syntax allows the closing `HEREDOC` to be indented but it won't strip space from the content so this works just fine.